.github/workflows/ci_nightly.yml

```
@@ -119,6 +119,16 @@ jobs:
       contents: read
       id-token: write
 
+  generate_blame_list:
+    name: Generate Blame List
+    runs-on: ubuntu-latest
```

(Move workflow changes to a separate PR)

---

Pin to a specific version: https://github.com/ROCm/TheRock/blob/main/docs/development/style_guide.md#pin-action-runs-on-labels-to-specific-versions
```suggestion
  generate_blame_list:
    name: Generate Blame List
    runs-on: ubuntu-24.04
```